addon/components/columns/base.js

```diff
@@ -73,6 +73,21 @@ const Column = Component.extend(DraggableColumnMixin, {
   sortIcons: null,
 
   /**
+   * @property sortIcon
+   * @type {String}
+   */
+  sortIcon: computed('column.sorted', 'column.ascending', function() {
+    let sorted = this.get('column.sorted');
+
+    if (sorted) {
+      let ascending = this.get('column.ascending');
+      return ascending ? 'iconAscending' : 'iconDescending';
+    }
+
+    return 'iconSortable';
+  }),
```

Comment:
```js
  /**
   * @property sortIcon
   * @type {String|null}
   * @private
   */
  sortIcon: computed('column.{sortable,sorted,ascending}', function() {
    let sorted = this.get('column.sorted');

    if (sorted) {
      let ascending = this.get('column.ascending');
      return ascending ? 'iconAscending' : 'iconDescending';
    }

    let sortable = this.get('column.sortable');
    return sortable ? 'iconSortable' : null;
  }),
```

I think I would rather mark `sortIcon` as `private` (for now). And only return a `sortIcon`, if it was set. That way later down in the template, we can just call `{{if sortIcon}}` as to not insert a "meaningless" `<i>` tag.

This behavior is slightly different than the current one, but I think this makes more sense. I doubt that the "empty" `<i>` tag is used in the wild.

@alexander-alvarez What is your opinion?